Makefile

```diff
@@ -9,28 +9,41 @@ OS=$(shell uname -s)
 EXAMPLE_DIRS=$(shell find ./examples -name '*.csproj' -exec dirname {} \;)
 TEST_DIRS==$(shell find ./test -name '*.csproj' -exec dirname {} \;)
 
-LINUX_FRAMEWORK=netcoreapp1.1
-DEFAULT_FRAMEWORK?=$(LINUX_FRAMEWORK)
+#framework can be change to net45 or similar
+FRAMEWORK?=netcoreapp1.1
+
+CONFLUENT_VERSION?=3.2
```

Comment:
Yes, i couldn't figure a proper way with oss-version and the way the url is generated... Perhaps add some comments